lib/bokken/periodically.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Bokken.Periodically do
+  use GenServer
+
+  alias BokkenWeb.NinjaController
+
+  def start_link(_opts) do
+    GenServer.start_link(__MODULE__, %{})
+  end
+
+  def init(state) do
+    schedule_work()
+    {:ok, state}
+  end
+
+  def handle_info(:work, state) do
+    NinjaController.notify_ninja_birthday()
+    schedule_work()
+    {:noreply, state}
+  end
+
+  defp schedule_work() do
+    Process.send_after(self(), :work, 24 * 60 * 60 * 1000)
```

Comment:
I don't like magic values. Make `24 * 60 * 60 * 1000` a macro called `@one_day`, for example.